src/Corporation/Actions.ts

```diff
@@ -198,125 +198,120 @@ export function acceptInvestmentOffer(corporation: Corporation): void {
   corporation.investorShares += investShares;
 }
 
-export function sellMaterial(material: Material, amount: string, price: string): void {
-  if (price === "") price = "0";
-  if (amount === "") amount = "0";
-  let cost = price.replace(/\s+/g, "");
-  cost = cost.replace(/[^-()\d/*+.MPe]/g, ""); //Sanitize cost
-  let temp = cost.replace(/MP/, "1.234e5");
+export function convertPriceString(price: string): string | number {
```

Comment:
Let's just always return a string. This is going to get eval'd later no matter what, there's no reason to have complicated types.